doc/releases/luminous.rst

```diff
@@ -1,3 +1,109 @@
+v12.2.8 Luminous
+================
+
+This is the eighth bug fix release of the Luminous v12.2.x long term stable
+release series. This release contains several bugfixes across all the components
+and we recommend all users upgrade.
+
+Upgrade Notes from previous luminous releases
+---------------------------------------------
+
+When upgrading from v12.2.5 or v12.2.6 please note that upgrade caveats from
+12.2.5 to 12.2.7 upgrades still apply to 12.2.8 as well.
+luminious-12-2-5-upgrades_ .Upgrades from 12.2.7 or earlier releases like 12.2.4
+or not running bluestore can proceed normally. 12.2.8 has fixes for scrub to fix
+the warnings generated from 12.2.6 corruption of data digest.
```

Comment:
Let's be more specific here.  New paragraph, and something like

In 12.2.7 we fixed the data digest regression from 12.2.6, but 12.2.7 clusters still generated health warnings like ```[ERR] 11.288 shard 207: soid
11:1155c332:::rbd_data.207dce238e1f29.0000000000000527:head data_digest
0xc8997a5b != data_digest 0x2ca15853
```.  12.2.8 improves the deep scrub code to automatically repair these inconsistencies.